specification/security_model.adoc

```diff
@@ -1,9 +1,316 @@
 [[security_model]]
-== Security and Threat Model
+== Security Model
+
+The CoVE-IO securiy model is built on the following assumptions:
+
+- Only the TVM owner can assess of a TDI trustworthiness. Based on that
+  assessment, it explicitly accepts or rejects a TDI into its TCB.
```

Comment:
we may want to explicitly state that - 
Until a TDI is accepted by the TVM: a.) no direct confidential memory access is allowed by the TDI, and b.) trusted MMIO access to the TDI is blocked by TSM. Untrusted MMIO access to the TDI may still be allowed by the VMM for ordinary VMs.

Reply:
Right. That's kind of what I was trying to describe later in that section but your prose is definitely clearer. I used it, thanks.